mllib/src/main/scala/org/apache/spark/ml/feature/StringIndexer.scala

```diff
@@ -131,6 +167,12 @@ object StringIndexer extends DefaultParamsReadable[StringIndexer] {
   private[feature] val KEEP_INVALID: String = "keep"
   private[feature] val supportedHandleInvalids: Array[String] =
     Array(SKIP_INVALID, ERROR_INVALID, KEEP_INVALID)
+  private[feature] val FREQ_DESC: String = "frequency_desc"
+  private[feature] val FREQ_ASC: String = "frequency_asc"
+  private[feature] val ALPHABET_DESC: String = "alphabet_desc"
+  private[feature] val ALPHABET_ASC: String = "alphabet_asc"
```

Comment:
@gatorsmile Thanks much for the suggestion. Changed them to lowerCamelCase. 
@felixcheung Any additional suggestions?